### PR TITLE
Add GIF and WebP support specification with Fresco 0.11

### PR DESCRIPTION
**Pull request by charpeni**
Motivation #8455
Fixes #8501

With a bonus fix typo !

screen shot 2016-07-25 at 14 16 01

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @nicklockwood to be potential reviewers.

File: Libraries/Image/Image.ios.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const ImageViewManager = NativeModules.ImageViewManager;
  * including network images, static resources, temporary local images, and
  * images from local disk, such as the camera roll.
  *
- * This exmaples shows both fetching and displaying an image from local storage as well as on from
+ * This examples shows both fetching and displaying an image from local storage as well as on from
```
Comment (JoelMarcey) [inline Libraries/Image/Image.ios.js]: You fixed 1/2. Can you fix the other 1/2? :)

This should say "This example"  -- corrected the misspelling, but it being plural was also wrong.

File: Libraries/Image/Image.ios.js [hunk 2]
```diff
@@ -92,6 +92,37 @@ const ImageViewManager = NativeModules.ImageViewManager;
  *   () => DisplayAnImageWithStyle
  * );
  * ```
+ *
+ * ### GIF and WebP support on Android
+ *
+ * By default on Android, GIF and WebP are not supported.
```
Comment (JoelMarcey) [inline Libraries/Image/Image.ios.js]: Maybe "By default, GIF and WebP are not supported on Android." ??

File: Libraries/Image/Image.ios.js [hunk 3]
```diff
@@ -92,6 +92,37 @@ const ImageViewManager = NativeModules.ImageViewManager;
...
+ *
+ * You will need to add some optional modules in `android/app/build.gradle` depending on the needs of your app.
```
Comment (JoelMarcey) [inline Libraries/Image/Image.ios.js]: Missing comma.

"....`android/app/build.gradle`, depending on the needs....."

File: Libraries/Image/Image.ios.js [hunk 4]
```diff
@@ -92,6 +92,37 @@ const ImageViewManager = NativeModules.ImageViewManager;
...
+ *   // For WebP support, including animated WebP
+ *   compile 'com.facebook.fresco:animated-webp:0.11.0'
+ *   compile 'com.facebook.fresco:webpsupport:0.11.0'
+ *
+ *   // For WebP support, without animations
+ *   compile 'com.facebook.fresco:webpsupport:0.11.0'
+ * }
+ * ```
+ *
+ * Also, if you use GIF with ProGuard you will need to add this rule in `proguard-rules.pro` :
```
Comment (JoelMarcey) [inline Libraries/Image/Image.ios.js]: Missing comma

"Also, if you use GIF with ProGuard, you will need...."

**Comment by JoelMarcey** [comment]
Seems reasonable to me, other than some minor requests for fixes. Thanks!

**Comment by charpeni** [comment]
Oops, you're totally right. 

Should be fixed now.

**Comment by mkonicek** [comment]
Thanks a lot! Didn't realize Fresco split its support for GIFs to a separate library. Good they split up WebP support, it's rarely used.

@facebook-github-bot shipit

**Pull request closed without merging**